packages/integrations/cloudflare/README.md

```diff
@@ -38,44 +38,45 @@ export default defineConfig({
 
 ## Options
 
-### Mode
+### `mode`
 
 `mode: "advanced" | "directory"`
 
 default `"advanced"`
 
-Cloudflare Pages has 2 different modes for deploying functions, `advanced` mode which picks up the `_worker.js` in `dist`, or a directory mode where pages will compile the worker out of a functions folder in the project root. For most projects the adapter default of `advanced` will be sufficient; the `dist` folder will contain your compiled project.
+Cloudflare Pages has 2 different modes for deploying functions, `advanced` mode which picks up the `_worker.js` in the `dist` folder, or a directory mode where pages will compile the worker out of a `functions` folder in the project root.
 
 #### `mode:directory`
```

Comment:
I'm not very happy with the format `mode:directory`, any other suggestions. I was thinking about `directory` or `directory mode`, but I don't find them perfect either.